bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
I suppose that as long as Arbiter sources are compiled with `-fexceptions`, it shouldn't matter.